melodies_monet/plots/surfplots.py

```diff
@@ -1250,7 +1250,7 @@ def scorecard_step4_GetRegionLUCDate(ds_name=None,region_list=None,datelist=None
             region_date_rural_here = []
             region_date_urban_here = []
             for i in range(len(region_here['Time'])):
-                date_here1 = region_here['Time'][i]
+                date_here1 = region_here['Time'].values[i]
```

Comment:
@btang1 can you confirm "values" can be added here? This seems like it should work fine, but was not in your original code.

Reply:
Seems fine to me too.

Reply:
@rschwant @zmoon I saw the change, I cannot say it works. we need to run and test, if test if past then we can change?
btw, who changed it?